python-checks/src/test/resources/checks/cdk/s3BucketVersioning.py

```diff
@@ -0,0 +1,42 @@
+from aws_cdk import aws_s3 as s3
+
+a = int(10)
+
+bucket = s3.Bucket(self, "MyUnversionedBucket", versioned=False) # Noncompliant
+                                              # ^^^^^^^^^^^^^^^
+
+# Noncompliant@+1 {{Make sure using unversioned S3 bucket is safe here. Omitting 'versioned=True' disables S3 bucket versioning. Make sure it is safe here.}}
+bucket = s3.Bucket(self, "MyUnversionedBucket")
+#        ^^^^^^^^^
+
+bucket_versioning = False
+bucket = s3.Bucket(self, "MyUnversionedBucket",
+                   versioned=bucket_versioning # Noncompliant
+                   # highlight `versioned=bucket_versioning` as 2st location
```

Comment:
You can remove this comment. It was used for implementation purposes only. In test cases, the highlighting is shown with the `^` comments.